### PR TITLE
Minimal rewrite of README to explain Flux v2

### PR DESCRIPTION
**Pull request by squaremo**
- Adjust the description so it talks about the whole system (Flux) as
   one thing
 - Describe what GitOps Toolkit is, and how it relates to Flux v2.

### CODE REVIEW

File: README.md
Comment (dholbach) [inline README.md:52]:
```suggestion
To get started with Flux, start [browsing the
```

**Comment by squaremo** [comment]
(I'll commit suggested changes for now and rebase later)

File: README.md
Comment (stefanprodan) [inline README.md:1]: I would call it Flux and update the fluxcd/flux readme and set there v1. See https://github.com/linkerd/linkerd2

File: README.md
Comment (stefanprodan) [inline README.md:10]: Given that Flux v2 can work with S3 buckets and Helm repositories only, should we reformulate "configuration in Git repositories"?

File: README.md
Comment (hiddeco) [inline README.md:62]: I think GitHub does this automatically? See, e.g: https://github.com/fluxcd/toolkit/#Components

Comment (squaremo) [reply README.md:62]: Oh yes, I forgot about that.

File: README.md
Comment (squaremo) [reply README.md:10]: Yes, if there's a good replacement term -- I'm reluctant to spin sentences out further for the sake of precision. I could stretch it to "Git repositories (and similar)" but it gets a bit vague.

Comment (squaremo) [reply README.md]: Turned it around, I think it covers your point and still reads OK now.

Comment (stefanprodan) [reply README.md]: Great!

File: README.md
Comment (squaremo) [reply README.md:1]: I think it's worth being explicit that it's a new thing, especially while fluxcd/flux is being maintained.

Comment (stefanprodan) [reply README.md:1]: Ok good point

**Comment by squaremo** [comment]
Rebass! :fish: I would think we'll wait until the repo is renamed to actually merge it.

**Review by hiddeco — APPROVED** [review]
:tada:

**Review by stefanprodan — APPROVED** [review]
Thanks @squaremo 🍰 

Please put this on hold until we rename the repo

**Comment by stefanprodan** [comment]
Ref: #388

**Pull request merged**
